packages/apps/reputation-oracle/server/src/database/database.module.ts

```diff
@@ -42,7 +42,7 @@ import { EscrowPayoutsBatchEntity } from '../modules/escrow-completion/escrow-pa
         typeOrmLoggerService.setOptions(
           loggerOptions && loggerOptions[0] === 'all'
             ? 'all'
-            : (loggerOptions as LoggerOptions) ?? false,
+            : ((loggerOptions as LoggerOptions) ?? false),
```

Comment:
@Dzeranov, probably, some linter ignore is needed for this line. It gets autoformatted.